### PR TITLE
virttest.utils_netperf: fix check netperf process

### PR DESCRIPTION
**Pull request by xutian**
use cmd_output_safe instead cmd_status_output to avoid output of
netperf client make netperf.is_target_running function failed when
netperf running in not quiet mode.

ID: 1198022

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

File: virttest/utils_netperf.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def __init__(self, address, netperf_path, md5sum="", netperf_source="",
     def is_target_running(self, target):
         if self.client == "nc":
             list_cmd = "wmic process where name='%s' list" % target
-            status, output = self.session.cmd_status_output(list_cmd)
+            output = self.session.cmd_output_safe(list_cmd, timeout=120)
```
Comment (ypu) [inline virttest/utils_netperf.py]: We need force this function not raise the ShellSession related error with try: exception here.

**Comment by ypu** [comment]
@xutian Hi It is looks good and will merge it after you rebase it.

**Comment by xutian** [comment]
@ypu 

thanks for your reminder, have re-base.

**Comment by ypu** [comment]
LGTM

Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by ypu** [comment]
Thanks @xutian and @CongSmile will merge it.

**Pull request merged**